tests/test_interactions.py

```diff
@@ -238,7 +243,9 @@ def test_forward(self):
                     for m in self.instance.modules()
                 )
                 if small_batch_size and has_batch_norm:
-                    logger.warning(f"Skipping test for shapes {hs}, {rs}, {ts}")
+                    logger.warning(
+                        f"Skipping test for shapes {hs}, {rs}, {ts} becuase too small batch size for batch norm",
```

Comment:
typo

Reply:
you wouldn't believe how often I typo `because`